code/controllers/subsystem/mapping.dm

```diff
@@ -853,6 +853,10 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 
 /datum/controller/subsystem/mapping/proc/lazy_load_template(template_key, force = FALSE)
 	RETURN_TYPE(/datum/turf_reservation)
+
+	if(Master.current_initializing_subsystem == SSearly_assets)
+		return // early assets creates all antag datums to collate a spritesheet of them
```

Comment:
I don't like this.
@Mothblocks 
We should allow this and instead prevent whatever is causing the root silly

Reply:
It depends on when antag datums are created in live game. I think they're only created right before assigning them, so we can move the logic there